moduleroot/CONTRIBUTING.md

```diff
@@ -148,70 +155,125 @@ Your bundle is complete!
 Use `bundle show [gemname]` to see where a bundled gem is installed.
 ```
 
-NOTE some systems may require you to run this command with sudo.
+NOTE: some systems may require you to run this command with sudo.
 
 If you already have those gems installed, make sure they are up-to-date:
 
 ```shell
 % bundle update
 ```
 
-With all dependencies in place and up-to-date we can now run the tests:
+## Running Tests
+
+With all dependencies in place and up-to-date, run the tests:
+
+### Unit Tests
+
+With the PDK (recommended):
+
+```shell
+% pdk test unit
```

Comment:
If this already works with the supported modules (I would not be surprised, but haven't tested) then the installation instructions above could be modified from `bundler` to `pdk`.

If it's not yet working, please file a PDK ticket in the PDK-378 epic.